pkg/kubelet/dockertools/kube_docker_types.go

```diff
@@ -17,8 +17,36 @@ limitations under the License.
 package dockertools
 
 import (
+	"fmt"
+	"time"
+
 	dockertypes "github.com/docker/engine-api/types"
 )
 
 // APIContainer is the container information returned by ContainerList
 type APIContainer dockertypes.Container
+
+// ContainerState is the dockertypes.ContainerState with time.Time timestamp
+type ContainerState struct {
+	dockertypes.ContainerState
+	StartedAt  time.Time
+	FinishedAt time.Time
+}
+
+// Container is the container information returned by InspectContainer with time.Time timestamp
+type Container struct {
+	dockertypes.ContainerJSON
+	Created time.Time
+	State   *ContainerState
```

Comment:
Another option is to add access methods for these special fields, e.g, `GetCreatedTime()`. The call sites will switch using these methods and will not care about the underlying implementation. What do you think?